ravenframework/Optimizers/acquisitionFunctions/AcquisitionFunction.py

```diff
@@ -191,8 +190,7 @@ def testGradients(self, bayesianOptimizer, nSamples):
     funGrad = nd.Gradient(evalMethod, step=0.0001, order=4)
     diffVector = np.empty(nSamples)
     limits = np.array(self._bounds)
-    sampler = LHS(xlimits=limits, criterion='cm', random_state=42)
```

Comment:
Hm, should random_state be bayesianOptimizer._seed instead of 42? (This is in the current code, but possibly we should fix it.)